apps/projects/views.py

```diff
@@ -294,49 +298,87 @@ def raise_exception(self):
 
     def get_context_data(self, **kwargs):
         """Append insights to the template context."""
+
         context = super().get_context_data(**kwargs)
+        idea_blueprints = {"BS", "IC"}
+        map_idea_blueprints = {"MBS", "MIC", "PB"}
```

Comment:
The participatory budgeting needs to be treated separately as well, there Proposal model is used. They also contribute to the idea count though.